test/e2e_generic/values.yaml

```diff
@@ -1,6 +1,4 @@
 experimental:
-  multiNamespaceMode:
-    enabled: true
```

Comment:
This will cause the test to fail I believe, why don't we use the new thing here?